subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -4614,6 +4617,28 @@ static void mayfly_xtal_stop(void *params)
 	DEBUG_RADIO_CLOSE(0);
 }
 
+#define DRV_NAME CONFIG_CLOCK_CONTROL_NRF5_K32SRC_DRV_NAME
```

Comment:
How this will behave on non-Nordic chips?

Reply:
Current upstream controller is nRF specific, the newer architecture in development separates vendor specifics in lower layer modules.

Reply:
Ok